examples/dummy_plugin/dummy_plugin/banner.py

```diff
@@ -0,0 +1,50 @@
+"""Example of using a plugin to inject a custom banner across various pages."""
+
+from typing import Optional
+
+from django.utils.html import format_html
+
+from nautobot.extras.choices import BannerClassChoices
+from nautobot.extras.plugins import PluginBanner
+
+
+def banner(context, *args, **kwargs) -> Optional[PluginBanner]:
+    """
+    Construct a custom PluginBanner, if appropriate.
+
+    - If not authenticated, no banner is displayed.
+    - On all authenticated UI views, the banner includes a greeting to the logged-in user.
+    - On object table views, the banner also reports on the model being listed in the table.
+    - On object detail views, the banner also reports on the object being viewed, and whether it's the changelog view.
+    """
+    if not context.request.user.is_authenticated:
+        # No banner if the user isn't logged in
+        return None
+
+    # Banner content greeting the user
+    content = format_html(
+        "<div>Dummy Plugin says “Hello, <strong>{}</strong>!” 👋</div>",
```

Comment:
Should we be using emojis in our code base? As this might say to the users we they can store this data but this doesnt work in MySQL.

Reply:
Emojis work in MySQL *if you have it configured properly*. :-) And this isn't database data in any case, so I think it's fine regardless. Thanks for asking though!

Reply:
> Should we be using emojis in our code base? As this might say to the users we they can store this data but this doesnt work in MySQL.

https://nautobot.readthedocs.io/en/stable/configuration/required-settings/#mysql-unicode-settings 😎